includes/ConsoleTasks/RegenerateStylesheetsTask.php

```diff
@@ -32,7 +32,7 @@ public function execute()
         foreach (['bootstrap-main', 'bootstrap-alt', 'bootstrap-auto'] as $file) {
             file_put_contents(
                 self::RESOURCES_GENERATED . '/' . $file . '.css',
-                $scss->compile('/*! Do not edit this auto-generated file! */ @import "' . $file . '";'));
+                $scss->compileString('/*! Do not edit this auto-generated file! */ @import "' . $file . '";'));
```

Comment:
Per comment on IRC, this looks to be missing the `->getCss()` call to actually retrieve the generated CSS - at the moment it seems to be generating an empty file.

```suggestion
                $scss->compileString('/*! Do not edit this auto-generated file! */ @import "' . $file . '";')->getCss());
```